packages/datadog_flutter_plugin/example/lib/main.dart

```diff
@@ -28,7 +28,10 @@ void main() async {
         printLogsToConsole: true,
       ),
       rumConfiguration: applicationId != null
-          ? RumConfiguration(applicationId: applicationId)
+          ? RumConfiguration(
+              applicationId: applicationId,
+              detectLongTasks: true,
```

Comment:
It would be nice for the README.md to also reflect these changes, although the parameter is optional.

Reply:
👍 Yeah, I'll add it.